packages/sdk-react/README.md

```diff
@@ -150,6 +150,12 @@ const { loading, theme, themeString, teamsUserCredential } = useTeamsUserCredent
 This hook function leverage `useData` to call Graph API. It will execute the fetchGraphDataAsync function that the developer passes in first.
 If user has not consented to the scopes of AAD resources, `useGraph()`, `useGraphWithCredential` will automatically call login function to pop up the consent dialog. So, developers can focus on the business logic of how to fetch Microsoft Graph data.
 
+### BaseDashboard
```

Comment:
@MuyangAmigo Please help to review the introduction of the two dashboard-related classes